### DISCUSSION TITLE
How to properly license generated content?

### DISCUSSION THREAD

**Discussion by WizardUli** [McShelby/hugo-theme-relearn#430] [Q&A] [answered]
I've been wondering how to properly license a generated website. I can't just slap Apache 2 or CA-BY 4 license with my copyright to the root of the generated content because there are a lot of 3rd party components like .js files and fonts.

I'm not even sure which files
- can be considered generated, i.e. are hugo+relearn output
- which are copied from relearn
- which are copied from 3rd parties

**Comment by McShelby** [comment, marked as answer]
As you can see, the theme itself ist [MIT licensed](https://github.com/McShelby/hugo-theme-relearn).

All 3rd party dependencies are listed on the [credits page](https://mcshelby.github.io/hugo-theme-relearn/more/credits/index.html). You can follow the links to find out their license.
